CHANGELOG.rst

```diff
@@ -11,7 +11,7 @@
 * Cope gracefully with a .pyc file with no matching .py file (`#2038`_). Thanks
   `@nedbat`_.
 
-*
+* Add hint to error message hinting possible missing __init__.py (`#478`_). Thanks `@DuncanBetts`_.
```

Comment:
You need to add the actual link to your user, see the lines below for `@nedbad_`:

```
.. _@DuncanBetts: https://github.com/DuncanBetts
```

(Add it before nedbat's to keep the list sorted)